src/rawrequest.schema.json

```diff
@@ -20,12 +20,54 @@
     "rawrequest": {
       "additionalProperties": false,
       "properties": {
+        "headers": {
+          "type": "object",
+          "description": "The headers of the request made to OpenWhisk/Petridish",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "properties": {
+            "x-openwhisk-activation-id": {
+              "type": "string",
+              "description": "OpenWhisk Activation ID"
+            },
+            "x-request-id": {
+              "type": "string",
+              "description": "Request ID generated by OpenWhisk"
+            },
+            "x-cdn-request-id": {
+              "type": "string",
+              "description": "Request ID generated by the CDN"
+            },
+            "x-backend-name": {
+              "type": "string",
+              "description": "Name of the Backend handling the request.",
+              "meta:enum": {
+                "localhost--F_Petridish": "Fake Backend name of the local Petridish server"
+              }
+            }
+          }
+        },
+        "method": {
+          "type": "string",
+          "description": "The HTTP method of the request made to OpenWhisk/Petridish. Note: OpenWhisk converts all methods to lowercase.",
+          "meta:enum": {
+            "get": "GET request",
+            "post": "POST request",
+            "put": "PUT request (not handled by Helix)",
+            "delete": "DELETE request (not handled by Helix)"
+          }
+        },
         "params": {
           "type": "object",
           "additionalProperties": {
             "type": "string"
           },
           "properties": {
+            "req": {
+              "type": "string",
+              "description": "A JSON-encoded map of request parameters"
```

Comment:
@tripodsan do we still need this? In production, the value is never used and in development it just introduces an inconsistency.

Reply:
we can drop this, since we now have `params`.